downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -35,24 +35,56 @@ Enter your Red Hat Registry Service Account credentials in `registry_username` a
 When `registry_url` is `registry.redhat.io`, username and password are required if not using a bundle installer.
```

Comment:
Similar to the above comment "the/your username and password are required if not using a bundle installer."